packages/kbn-test/src/functional_test_runner/lib/mocha/reporter/scout_ftr_reporter.ts

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
This reporter currently does not log skipped tests, would be good to include them (the Playwright report contains skipped tests).

Reply:
I'll look into this and see what hook I've missed. 🤔

Reply:
Fixed in [dc193ee](https://github.com/elastic/kibana/pull/202906/commits/dc193ee299c7bb44e4c676ee6f6c6944a5cc47fe)